src/components/EncounterPage/EncounterPage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import EnemyEntry from './EnemyEntry'
+import PlayerEntry from './PlayerEntry'
+import PageButton from '../Buttons/PageButton'
+import styled from 'styled-components'
+
+export default function EncounterPage({ creatureEntries }) {
+  const creaturesByInitiative = creatureEntries
+    .slice()
+    .sort((entry1, entry2) => entry1.initiative < entry2.initiative)
+
+  return (
+    <>
+      {creaturesByInitiative.map((creatureEntry, index) =>
+        creatureEntry.enemyName ? (
+          <EnemyEntry
+            enemyName={creatureEntry.enemyName}
+            armorClass={creatureEntry.armorClass}
+            HP={creatureEntry.HP}
+            initiative={creatureEntry.initiative}
+            key={index}
```

Comment:
index for key value is not good. you could put something unique in there. is the enemyName unique? otherwise use uuid.

Reply:
Ah right, I was going to use uuid and then forgot about it. Good catch, thanks for pointing that out!